pkg/config/runtime.go

```diff
@@ -6,4 +6,5 @@ type RuntimeConfig struct {
 	RedirectURI    string
 	GlobalCodeMode bool
 	WorkingDir     string
+	ModelOverrides []string
```

Comment:
It looks weird to me that this is in the RuntimeConfig, the runtime doesn't need or use this